railgun/test/integration/suite_test.go

```diff
@@ -171,7 +199,14 @@ func TestMain(m *testing.M) {
 		}
 	}
 
-	fmt.Println("INFO: testing environment is ready, running tests")
+	fmt.Printf("INFO: running final testing environment checks")
+	serverVersion, err := env.Cluster().Client().ServerVersion()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error: could not retrieve server version for cluster: %s", err.Error())
```

Comment:
```suggestion
		fmt.Fprintf(os.Stderr, "Error: could not retrieve server version for cluster: %w", err)
```

Reply:
The `%w` doesn't make sense in this context: there's no caller to handle the error nevertheless unwrap it.

Reply:
not a blocker

Reply:
It's a style thing: `%w` is equivalent to `%v` per `fmt` spec and my style preference is that using `%v` is better than `%s` + `String()`/`Error()` for brevity.

Optional.

Reply:
It's funny how often these formatting directives come up in PR reviews :joy: 

Ultimately, changing this will have basically no effect on the result, however if we can start just doing things one standard way that could be helpful to reduce these kinds of things in PR review so I went ahead and changed it to try and move towards a conformance of style as you suggest:

https://github.com/Kong/kubernetes-ingress-controller/pull/1558/commits/f7aa06e20578872cefd4eb743b93228e3faec0c9

Reply:
So as it turns out, trying to use `%w` in `Fprintf` will not compile:

```
test/integration/suite_test.go:138:4: Fprintf call has error-wrapping directive %w
```

We literally _can't_ do this.